src/modules/user-interaction/index.ts

```diff
@@ -0,0 +1,71 @@
+import {
+    USER_LIKE_ENTITY,
+    USER_UN_lIKE_ENTITY,
+    USER_DIS_LIKE_ENTITY
+} from '../../mutations/';
+import {
+    UserInteractionInput,
+    ResponseUserLikeEntity,
+    ResponseUserUnLikeEntity,
+    ResponseUserDislikeEntity,
+    UserInteraction,
+    EntityInteraction
+} from '../../types/';
+import { ClientType } from '../../index';
+import { GET_USERS_INTERACTIONS_QUERY, GET_USER_INTERACTION_QUERY } from '../../queries';
+export class UsersInteractions {
+
+    constructor(protected client: ClientType) {}
+
```

Comment:
Suggestion: Format this file pls

Reply:
I don't have formmater, can you send me one